src/components/calcite-tip-manager/usage/basic.md

```diff
@@ -0,0 +1,43 @@
+#### Basic
+
+Renders a tip manager using tip group as well as single tips.
```

Comment:
How does `Renders a tip manager using a group of tips and single tips.` sound?

Reply:
sounds good! i'll change it